leshan-server-core/src/main/java/org/eclipse/leshan/server/impl/LwM2mRequestSenderImpl.java

```diff
@@ -94,4 +94,13 @@ public void stop() {
             ((Stoppable) defaultRequestSender).stop();
         }
     }
+
+    @Override
+    public void cancelPendingRequests(Client client) {
+        if (client.usesQueueMode()) {
+            queuedRequestSender.cancelPendingRequests(client);
+        } else {
+            defaultRequestSender.cancelPendingRequests(client);
+        }
+    }
```

Comment:
Should not be a problem as `QueuedRequestSender` has its own instance of `CaliforniumLwM2mRequestSender`.